include/hw_bridge.h

```diff
@@ -109,17 +109,15 @@ enum canID_t {
 namespace ARM {
 namespace ACTUATOR {
 enum actuatorID_t { TURNTABLE, SHOULDER, ELBOW, WRISTLEFT, WRISTRIGHT, CLAW };
-std::string stringifyActuatorID(actuatorID_t actuator);
 }  // namespace ACTUATOR
 namespace PID {
-typedef struct __attribute__((__packed__)) {
+struct __attribute__((__packed__)) tuningApiPayload {
```

Comment:
compiler says typedef does nothing and throws an error

Reply:
Whats the error?

Reply:
To map the typdef to something you need to add a name after the closing '}', without it it does nothing. Ex `typedef uint8_t byte` vs `typedef uint8_t`. The name before the opening '{' is a tag not a type.

Reply:
i just misused typedef. alex explained it.